packages/patternfly-4/react-core/src/demos/BulkSelectTable.md

```diff
@@ -0,0 +1,225 @@
+    return (
+      <Pagination
+        itemCount={total}
+        page={page}
+        perPage={perPage}
+        onSetPage={(_evt, value) => {
+          this.fetch(value, perPage);
+        }}
+        onPerPageSelect={(_evt, value) => {
+          this.fetch(1, value);
+        }}
+        variant={variant}
+      />
+    );
+  }
+
+  buildSelectDropdown() {
+    const { isDropDownOpen, selectedItems, total } = this.state;
+    const numSelected = selectedItems.length;
+    const allSelected = numSelected === total;
+    const anySelected = numSelected > 0;
+    const isChecked = allSelected ? true : numSelected > 0 ? null : false;
+
+    const items = [
+      <DropdownItem key="item-1" onClick={() => this.handleSelectClick('none')}>
+        Select none (0 items)
+      </DropdownItem>,
+      <DropdownItem key="item-2" onClick={() => this.handleSelectClick('page')}>
+        Select page ({this.state.perPage} items)
+      </DropdownItem>,
+      <DropdownItem key="item-3" onClick={() => this.handleSelectClick('all')}>
+        Select all ({this.state.total} items)
+      </DropdownItem>
+    ];
+
+    return (
+      <Dropdown
+        onSelect={this.onDropDownSelect}
+        position={DropdownPosition.left}
+        toggle={
+          <DropdownToggle
+            splitButtonItems={[
+              <DropdownToggleCheckbox
+                id="example-checkbox-2"
+                key="split-checkbox"
+                aria-label={anySelected ? 'Deselect all' : 'Select all'}
+                isChecked={isChecked}
+                onClick={() => {
+                  anySelected ? this.handleSelectClick('none') : this.handleSelectClick('all');
+                }}
+              ></DropdownToggleCheckbox>
+            ]}
+            onToggle={this.onDropDownToggle}
+          >
+            {numSelected !== 0 && <React.Fragment>{numSelected} selected</React.Fragment>}
+          </DropdownToggle>
+        }
+        isOpen={isDropDownOpen}
+        dropdownItems={items}
+      />
+    );
+  }
+
+  renderToolbar() {
+    return (
+      <Toolbar className="pf-l-toolbar pf-u-justify-content-space-between pf-u-mx-xl pf-u-my-md">
+        <ToolbarGroup>
+          <ToolbarItem className="pf-u-mr-md">{this.buildSelectDropdown()}</ToolbarItem>
+        </ToolbarGroup>
+        {this.renderPagination()}
+      </Toolbar>
+    );
+  }
+
+  render() {
+    const { loading, res } = this.state;
+    const rows = res.map(post => ({
+      cells: [post.title, post.body],
+      selected: post.selected
+    }));
+
+    return (
+      <React.Fragment>
+        {this.renderToolbar()}
+        {!loading && (
+          <Table
+            header={<div></div>}
+            cells={['Title', 'Body']}
+            rows={rows}
+            onSelect={this.onSelect}
+            canSelectAll={false}
+          >
+            <TableHeader />
+            <TableBody />
+          </Table>
+        )}
+        {loading && (
+          <center>
```

Comment:
Can we position this in a patternfly way, like maybe `<div className="pf-l-bullseye">` or something like that?